.github/workflows/pr-review.yml

```
@@ -0,0 +1,56 @@
+      security-events: write
+      actions: read
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v3
+        with:
+          languages: 'javascript-typescript'
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)